directory_header_footer/context_processors.py

```diff
@@ -60,4 +60,5 @@ def header_footer_context_processor(request):
     return {
         'header_footer_contact_us_url': settings.HEADER_FOOTER_CONTACT_US_URL,
         'header_footer_css_active_classes': active_classes,
+        'header_footer_language_select': settings.HEADER_FOOTER_LANGUAGE_SELECT
```

Comment:
for consistency with other feature flags, please rename to something like `FEATURE_FLAG_IS_LANGUAGE_SELECTOR_ENABLED`

The `FEATURE_FLAG_` prefix is the important bit